iModelJsNodeAddon/api_package/ts/package.json

```diff
@@ -21,12 +21,14 @@
     "@itwin/core-common": "rc",
     "@itwin/core-geometry": "rc",
     "@itwin/eslint-plugin": "rc",
+    "@types/better-sqlite3": "^7.6.5",

```

Comment:
Is `IModelJsNative.SqliteDb` missing something you need, to instead motivate adding this new dev dependency?